app/assets/javascripts/sequence_show.js

```diff
@@ -512,3 +535,33 @@ function init_sequence_show(data) {
         }
     }
 }
+
+function toggle_column(col) {
```

Comment:
Im not sure what you mean with global namespace in this sentence.


_[Original comment](https://github.ugent.be/unipept/unipept/pull/528#discussion_r3472) by calmalak on Mon Nov 09 2015 at 22:24._